crates/store/re_video/src/decode/mod.rs

```diff
@@ -180,4 +179,5 @@ pub enum YuvRange {
 pub enum YuvMatrixCoefficients {
     Bt601,
     Bt709,
+    Identity,
```

Comment:
Maybe put this first (its the simplest, after all), and add a docstring for it

Reply:
right, did so on re_renderer as well so let's do that here!